generators/cleanup.js

```diff
@@ -120,10 +120,17 @@ function cleanupOldFiles(generator) {
             generator.removeFile(`${ANGULAR_DIR}shared/login/login.component.html`);
             generator.removeFile(`${ANGULAR_DIR}core/auth/user-route-access-service.ts`);
             generator.removeFile(`${ANGULAR_DIR}tsconfig.base.json`);
+            generator.removeFile(`${ANGULAR_DIR}webpack/webpack.common.js`);
+            generator.removeFile(`${ANGULAR_DIR}webpack/webpack.dev.js`);
+            generator.removeFile(`${ANGULAR_DIR}webpack/webpack.prod.js`);
```

Comment:
`webpack/utils.js` is no more needed and needs also to be deleted and cleaned up